content/en/getting_started/devsecops/_index.md

```diff
@@ -134,7 +134,7 @@ The Infrastructure DevSecOps bundles combine infrastructure monitoring with the
 {{< tabs >}}
 {{% tab "Infrastructure DevSecOps Pro" %}}
 
-Infrastructure DevSecOps Pro includes [Containers][1], [Serverless][2], and [CSM Pro][3]. It also includes more than [800+ out-of-the-box integrations][4].
+Infrastructure DevSecOps Pro includes [Containers][1], [Serverless][2], and [CSM Pro][3]. It also includes more than [{{< translate key="integration_count" >}}+ out-of-the-box integrations][4].
```

Comment:
```suggestion
Infrastructure DevSecOps Pro includes [Containers][1], [Serverless][2], and [CSM Pro][3]. It also includes more than {{< translate key="integration_count" >}} [out-of-the-box integrations][4].
```

Reply:
Our docs tool doesn't seem to know how to render a shortcode inside a link. I think the easiest way around this is probably to move the number outside of the link (and I also removed the plus sign, which seemed redundant with "more than" directly preceding the number).